ibis/backends/tests/test_array.py

```diff
@@ -401,17 +402,21 @@ def test_array_slice(backend, start, stop):
         param({"a": [[1, 2], [4]]}, {"a": [[2, 3], [5]]}, id="no_nulls"),
     ],
 )
-def test_array_map(backend, con, input, output):
+def test_array_map(con, input, output):
     t = ibis.memtable(input, schema=ibis.schema(dict(a="!array<int8>")))
     expected = pd.DataFrame(output)
 
     expr = t.select(a=t.a.map(lambda x: x + 1))
     result = con.execute(expr)
-    backend.assert_frame_equal(result, expected)
+    assert frozenset(map(frozenset, result["a"])) == frozenset(
```

Comment:
Converting to sets seems a bit weird. The array element orders are different? Also, instead of converting the column to a frozenset couldn't be sorting used?